### DISCUSSION TITLE
Naming of ports in wiring chart

### DISCUSSION THREAD

**Discussion by guerda** [jomjol/AI-on-the-edge-device#2018] [Q&A]
Dear all,

I'd like to flash my ESP with AI on the edge. Right now, I can't determine which ports are connected to which UART ports, if I look at the [documentation](https://jomjol.github.io/AI-on-the-edge-device-docs/Installation/#wiring) and the wiring chart:
![Shown wiring for flashing](https://jomjol.github.io/AI-on-the-edge-device-docs/img/wiring.png)

Can anybody help me to describe which port is which on the ESP? 

My ESP seems to have the ports different than shown in the wiring chart, as power and ground are the first in the top left instead of bottom left.

My order on the right side is the following:
* 3V3
* IO16
* IO0
* GND
* VCC
* U0R
* U0T
* GND/R

Thanks in advance!

**Comment by caco3** [comment]
Looking on https://microcontrollerslab.com/esp32-cam-ai-thinker-pinout-gpio-pins-features-how-to-program/ it should be the pins IO1 and IO3. They are sometimes also labeled as UOT and IOR
![grafik](https://user-images.githubusercontent.com/1783586/218333805-3850f693-3117-4a46-a2dd-69b93c7147af.png)
